### PR TITLE
178131341 remote trigger self destruct

### PR DESCRIPTION
**Pull request by androidmaven**

### CODE REVIEW

File: app/src/main/java/com/hover/stax/navigation/AbstractNavigationActivity.java [hunk 1]
```diff
@@ -82,15 +82,16 @@ protected NavController getNavController() {
     }
 
     public void checkPermissionsAndNavigate(int toWhere) {
```
Comment (davkutalek) [inline app/src/main/java/com/hover/stax/navigation/AbstractNavigationActivity.java:84]: This looks wrong, why is permissions check commented?

**Comment by davkutalek** [comment]
Don't forget to put bubble showcase back before merging

File: app/src/main/java/com/hover/stax/navigation/AbstractNavigationActivity.java (hunk 1 above)
Comment (androidmaven) [reply app/src/main/java/com/hover/stax/navigation/AbstractNavigationActivity.java]: Thanks, was an oversight to refractor back.

**Comment by alexosugo** [comment]
Closing this as it has been merged ti development.

**Pull request closed without merging**